sources/osgViewer/webgl-utils.js

```diff
@@ -110,7 +115,7 @@ var WebGLUtils = function () {
         /** function:(msg) */
         opt_onError ) {
         function handleCreationError( msg ) {
-            if ( msg.indexOf( 'WebGL2' ) !== -1 ) return;
+            if ( !doReportCreationError ) return;
```

Comment:
I tested it and it works :)

"doReportCreationError"  is "true" by default and if we test WebGL2, we set it back to "true" once we unsuccessfully tried WebGL2 (https://github.com/cedricpinson/osgjs/blob/c588a09269cee85ef9a4d3264a9b5e8cf2916287/sources/osgViewer/webgl-utils.js#L182)

Parameter to function won't work because handleCreationError is a callback